packages/terra-consumer-nav/package.json

```diff
@@ -36,8 +36,14 @@
     "terra-button": "^1.1.0",
     "terra-icon": "^1.0.0",
     "terra-mixins": "^1.0.0",
+    "terra-modal": "^1.2.0",
```

Comment:
now that we are consuming `terra-modal` can we remove 
https://github.com/cerner/terra-consumer/tree/Navigation-help-profile/packages/terra-consumer-nav/src/components/modal